### DISCUSSION TITLE
Relative links point to root?

### DISCUSSION THREAD

**Discussion by acidtone** [sveltejs/kit#4977] [Q&A] [answered]
This maybe a noob question but I can't figure out why my relative links are redirecting to the web root?

[GitHub repo](https://github.com/acidtone/tonygrimes.com/)
[Test page](https://serene-cajeta-eeb775.netlify.app/journal)
- click a journal link to receive a 404.

**Problem**: Relative links in a `/journal/index.svelte` page component resolve relative to the root directory:
![sk-relative-before](https://user-images.githubusercontent.com/6174466/168961330-a08614ac-b730-4215-86fb-0ca6ed81a8a6.jpg)

Clicking a journal post resolves to a 404

![sk-relative-after](https://user-images.githubusercontent.com/6174466/168961385-7fd2172d-b987-4182-ac42-6c47e110a42e.jpg)

**Questions**
- Is SK overriding the address that the browser is displaying?
- Is there a way to link to the current directory without hard-coding it?

**Comment by mrkishi** [comment, marked as answer]
Relative links from the `/journal` page will start at `/`. It sounds like you want to serve the page from `/journal/` instead, in which case you'd need to set the `trailingSlash` option to `always`.

**Reply by acidtone** [reply]
Thanks! That's exactly what I was looking for.
